tests/rbac/common/addresser/role_admin_test.py

```diff
@@ -78,18 +78,16 @@ def test_address_random(self):
             addresser.address_is(rel_address2), addresser.AddressSpace.ROLES_ADMINS
         )
 
-    def test_address_static(self):
-        """Tests address makes the expected output given a specific input"""
-        role_id = "99968acb8f1a48b3a4bc21e2cd252e67"
-        user_id = "966ab67317234df489adb4bc1f517b88"
-        expected_address = (
-            "bac00100005555326a1713a905b26359fc8da23333eee7570f3f6f7d2c1635f6deea00"
-        )
-        rel_address = addresser.role.admin.address(
-            object_id=role_id, related_id=user_id
-        )
-        self.assertIsAddress(rel_address)
-        self.assertEqual(rel_address, expected_address)
-        self.assertEqual(
-            addresser.address_is(rel_address), addresser.AddressSpace.ROLES_ADMINS
-        )
+    def test_addresser_parse(self):
+        """Test addresser.parse returns a parsed address"""
+        role_id = addresser.role.unique_id()
+        user_id = addresser.user.unique_id()
+        rel_address = addresser.role.admin.address(role_id, user_id)
+        parsed = addresser.parse(rel_address)
```

Comment:
minor style quibble -- can you put a line or two between the setup methods and the line that actually executes the test? It helps in reading as a Given/When/Then format

Reply:
resolved